src/main/java/net/minestom/server/utils/crypto/KeyUtils.java

```diff
@@ -0,0 +1,47 @@
+package net.minestom.server.utils.crypto;
+
+import java.nio.charset.StandardCharsets;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.X509EncodedKeySpec;
+import java.util.Base64;
+
+public final class KeyUtils {
+    private static final Base64.Encoder MIME_ENCODER = Base64.getMimeEncoder(76, "\n".getBytes(StandardCharsets.UTF_8));
+    private static final String RSA_HEADER = "-----BEGIN RSA PUBLIC KEY-----\n";
+    private static final String RSA_FOOTER = "\n-----END RSA PUBLIC KEY-----\n";
+
+    public enum SignatureAlgorithm {
+        SHA256withRSA,
+        SHA1withRSA
+    }
+
+    public enum KeyAlgorithm {
+        RSA
+    }
```

Comment:
Thinking about it, these enums don't really even need to exist. These are only used in the string constant context, i.e. by calling `name()` all the time, so they could just be string constants instead of enums.

Reply:
With the enum you can be sure that the method will exists, but with a string you might accidentally pass a wrong value.

Reply:
That is true, but is there even even that anyone other than us internally could pass an incorrect value? Because if there is not (and I'm pretty sure there isn't), we don't need the enum to guard against that.